### DISCUSSION TITLE
在.\icarus\include\style中做的修改发布后没有在网页中体现

### DISCUSSION THREAD

**Discussion by Nanzhan-17** [ppoffice/hexo-theme-icarus#1051] [Q&A]
在.\icarus\include\style中设置了tag颜色、黏性目录等样式，hexo s的时候是正常的，但是hexo d之后并没有成功体现在发布的博客上，有人知道为什么吗

**Comment by Nanzhan-17** [comment]
我发现我对于style的更改，在本地文件的public\css\default中可以体现出来，但是发布后却在网页端体现不出来

**Comment by Nanzhan-17** [comment]
在我修改了style文件夹里面的文件后，我hexo s预览的F12里面的default.css和hexo d发布后的default.css不一样，发布后的没有我对样式做出的修改

**Comment by ppoffice** [comment]
清空下浏览器缓存或者用隐私模式打开看看

**Reply by Nanzhan-17** [reply]
尝试过了，没有用。另外我通过npm下载的Icarus5.0.0没有_config.post.yml文件请问是为什么呢

**Comment by ppoffice** [comment]
`hexo clean`了么

**Reply by Nanzhan-17** [reply]
clean过了，也没有用，我通过把hexo s预览界面F12中的default.css全部复制到public/css/default.css中解决了问题，但是我依然想问一下我为什么没有_config.post.yml呢

**Reply by ppoffice** [reply]
自己新建

**Reply by Nanzhan-17** [reply]
> 自己新建

在_config.icarus.yml同目录下新建一个名为_config.post.yml的文件就可以了吗，新建之后要怎么引用啊，需要额外的部署吗
